types/src/on_chain_config/execution_config.rs

```diff
@@ -195,6 +215,16 @@ pub struct ExecutionConfigV6 {
     pub gas_price_to_burn: u64,
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Eq, Serialize)]
+pub struct ExecutionConfigV7 {
+    pub transaction_shuffler_type: TransactionShufflerType,
+    pub block_gas_limit_type: BlockGasLimitType,
+    pub enable_per_block_gas_limit: bool,
+    pub transaction_deduper_type: TransactionDeduperType,
+    pub gas_price_to_burn: u64,
+    pub persisted_auxiliary_info_version: u8,
```

Comment:
Silly question: why is this a version and not something simpler, e.g., like a bool?

Reply:
In case new variants are added to `PersistedAuxiliaryInfo` in the future, and we want all the validators to switch to creating the new variant simulataneously, we can increment this version here without adding a new feature flag.